src/taco-cli/package.json

```diff
@@ -42,6 +42,7 @@
         "rimraf": "2.2.6",
         "semver": "^4.3.6",
         "taco-dependency-installer": "file:../taco-dependency-installer",
+        "taco-tests-utils": "file:../taco-tests-utils",
```

Comment:
Also we should double check that it is correctly picked up by updateDynamicDependencies.js to make sure these tests don't fail if we build the sources on one computer, transfer them to a new computer and then try to run the tests